drivers/edac/edac_ibecc.c

```diff
@@ -18,6 +18,9 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(edac_ibecc, CONFIG_EDAC_LOG_LEVEL);
 
+#define DEVICE_NODE DT_NODELABEL(ibecc)
+#define PCI_ENDPOINT PCIE_BDF(0, 0, 0)
```

Comment:
Is this BDF value treated in some special way by `pcie_probe()`, e.g. to mean "any BDF"? I don't see it documented in `include/drivers/pcie/pcie.h` - perhaps it should be? In that case I'd also propose to add this as a new `PCIE_BDF_ANY` define to `pcie.h`.

Reply:
Oh, I see `pcie_probe()` accepts `PCIE_ID_NONE` for the ID parameter, so perhaps the appropriate analogue for the BDF would be `PCIE_BDF_NONE`.

Reply:
This is Host Bridge BDF, I can rename `PCI_ENDPOINT` to `PCI_HOST_BRIDGE`.

Reply:
Oh, ok. So there's no PCIe device strictly for IBECC, rather the information about it is found under the host bridge PCIe device?

Reply:
Yes